networks/debug_networks.py

```diff
@@ -0,0 +1,120 @@
+"""This module contains simple, easy to debug networks that have the same interface as the other
+networks in the package.
+"""
+
+import torch
+from torch import nn
+
+from .inference_net import to_phase_space
```

Comment:
Probably this doesnt work any more

Reply:
How should we do this?

Reply:
Not sure... Maybe inherit from the transformer so the class also has it?

Reply:
I just realized that these networks are all not working anymore since I refactored the real ones. I'll make new debug nets that extend the real ones, but do not call their `__init__` and `forward()`.